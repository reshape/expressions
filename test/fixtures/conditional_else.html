<p>x</p>
<if condition="false">
  <p>hey: {{ foo }}</p>
</if>
<else>
  <p>{{ foo }}</p>
</else>
<p>x</p>
