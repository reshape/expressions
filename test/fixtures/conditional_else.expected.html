<p>x</p>

  <p>bar</p>
<p>x</p>
